app/courses/page.js

```diff
@@ -0,0 +1,41 @@
+import Link from "next/link";
```

Comment:
if it's possible to import this thing in a root file and all the child file can use it, kindly do it. if not possible, leave it as it is

Reply:
I have checked and found that it's not possible to use Link in child files without explicitly importing it in each file, as Next.js doesn't support global imports for components.